doc/testing.md

```diff
@@ -24,21 +24,27 @@ multiple_choice :question_1 do
   option :A
   option :B
 
-  next_node :question_2
+  next_node do
+    question :question_2
+  end
```

Comment:
Although I don't think there's currently anything forcing us to do this, I agree that it's a standard that has been adopted in the code and the documentation should reflect it. However, since it was already missing, I think it's out of scope for this PR.

I've opened a new PR to address this: #2392